docs/reference/dockerd.md

```diff
@@ -124,6 +124,7 @@ type `dockerd`.
 To run the daemon with debug output, use `dockerd --debug` or add `"debug": true`
 to [the `daemon.json` file](#daemon-configuration-file).
 
+> [!NOTE]
```

Comment:
This one didn't have a **Note** before, but it seems fine/appropriate to add one. Just thought I'd call it out